src/pe/debug.rs

```diff
@@ -100,7 +100,7 @@ impl ImageDebugDirectory {
         opts: &options::ParseOptions,
     ) -> error::Result<Self> {
         let rva = dd.virtual_address as usize;
-        let offset = utils::find_offset(rva, sections, file_alignment, opts).ok_or_else(|| {
+        let offset = utils::find_offset(rva, sections, file_alignment, opts).map_err(|_| {
```

Comment:
Yes it's true. I did it because I thought than this error message is more representative. Maybe I could combine these two messages using `map_err`? Or just leave what find_offset returns?

Reply:
I would simply propagate the error `find_offset()` potentially returns.

It is probably more informative to nest the error of `find_offset()` inside a more generic error that adds context, but that requires something like the `anyhow` crate. It is a good idea to do that, but it's better to fix things one at a time.